app/Helpers/render/game.php

```diff
@@ -50,10 +50,10 @@ function gameAvatar(
     }
 
     return avatar(
-        resource: 'game',
+        resource: $isHub ? 'hub' : 'game',
         id: $id,
         label: $label !== false && ($label || !$icon) ? $label : null,
-        link: $href ?? route('game.show', $id),
+        link: $isHub ? route('hub.show', ['gameSet' => $id]) : route('game.show', $id),
```

Comment:
When `$isHub`, it should change the tooltip too.

[game=]
![image](https://github.com/user-attachments/assets/c451f458-10b8-4ff4-83a2-135494296b53)

[hub=]
![image](https://github.com/user-attachments/assets/2d1a0229-483f-403f-ac88-dd6409a1f973)

Reply:
This may be a bigger issue. I think what we really want to do is convert [game=31277] to [hub=28019] on submit.

I've modified this to happen in latest (we'll probably need to run our game -> hub shortcode convert command again on deploy).

Preview is still affected by this issue, but not sure if that's worth hunting down.